test/integration/admin_active_stats_test.js

```diff
@@ -0,0 +1,122 @@
+// This file contains helper functions for writing JavaScript unit tests, in the
```

Comment:
I think it makes sense to move the non-c++ test files to a subdir here also.

Reply:
done